membership_withdrawal/views/membership_withdrawal_reason_view.xml

```diff
@@ -29,7 +31,7 @@
                 <filter
                     string="Archived"
                     domain="[('active','=', False)]"
-                    name="archived_reasons"
```

Comment:
Don't rename this for easing migration of third-party modules. It's not bad name at all.